src/components/GitHubIssueLink/GitHubIssueLink.module.css

```diff
@@ -0,0 +1,14 @@
+.githubLink {
+  font-style: italic;
+  box-sizing: border-box;
+  min-width: 0;
+  transition: color var(--t-interaction);
+  color: var(--ifm-font-color-base);
+  text-align: right;
```

Comment:
I take it back. I think a better design choice is to left-align it (removing this style) and give it negative margin (-25px) so that it hugs the H1:
<img width="996" alt="image" src="https://github.com/user-attachments/assets/f4b153c6-c7d4-496f-9d13-08240912d949" />
<img width="1400" alt="image" src="https://github.com/user-attachments/assets/bd633573-517e-4401-ab5f-c6678a004894" />